README.md

```diff
@@ -12,3 +12,4 @@ For contributing to this repository, please follow the guidelines specified in t
 
 
 
+
```

Comment:
What's with the newline here?

Reply:
Trying to adjust the old hugo hierarchy, added by mistake this line. I'll remove this :)